### PR TITLE
Update tools versions

### PR DESCRIPTION
**Pull request by voron**
* Bump gosu, helm, helmfile, sops, yq version to latest ones
* Remove helm-secrets version pin

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -76,7 +76,7 @@ RUN . /.versions && \
 ## Install plugins (already as the specified user)
 RUN sudo -iu kubectl bash -c 'set -e; \
       helm plugin install https://github.com/databus23/helm-diff; \
-      helm plugin install https://github.com/jkroepke/helm-secrets --version v3.12.0; \
+      helm plugin install https://github.com/jkroepke/helm-secrets; \
```
Comment (VladStarr) [inline Dockerfile:79]: WDYT on pinning plugin versions and configuring them via `.versions`?

**Review by VladStarr — COMMENTED** [review]
Just one question :)

File: Dockerfile (hunk 1 above)
Comment (voron) [reply Dockerfile:79]: Version pinning is the correct option to get reproducible builds for sure, but I think we may get back to it when needed. I'm unaware of using helm from this image in our projects.

**Review by VladStarr — APPROVED** [review]
LGTM

**Pull request merged**